CONTRIBUTING.md

```diff
@@ -49,6 +49,8 @@ version 4.1.0. **[You've followed all of the
 best practices for writing high quality definitions,](https://github.com/flow-typed/flow-typed/issues/13#issuecomment-214892914)**
 and now you'd like to contribute it:
 
+> Flow-typed comes with a handy script to simplify the file creation process `./create_def.sh` which takes two args, the library name and the version you'd like to type. In the :point_down: example that would be `./create_def.sh left-pad 4.x.x`.
```

Comment:
This'll need to be updated :)

Reply:
Oh yep, let me fix this later today

Reply:
Actually I think it's fine as is. It just proxies to call `node cli/dist/cli.js create-def left-pad 4.x.x` so docs with how to contribute are still valid